### DISCUSSION TITLE
Property 'prisma' does not exist on type 'Global & typeof globalThis'.ts(2339)

### DISCUSSION THREAD

**Discussion by LGmatrix13** [prisma/prisma#6374] [Q&A] [answered]
VS code is repeating this error everywhere

I am using this example: https://github.com/prisma/prisma-examples/tree/latest/typescript/rest-nextjs-api-routes

A particular example is this API route. The word "post" is highlighted as invalid.

```
import type { NextApiRequest, NextApiResponse } from 'next'
import prisma from '../../../lib/prisma'


export default async function handle(req: NextApiRequest, res: NextApiResponse) {
  const posts = await prisma.post.findMany({
    take: 12,
    where: { published: true },
    include: { user: true },
  })
  res.json(posts)
}
```

**Comment by LGmatrix13** [comment, marked as answer]
Oops, just a typescript error. Was able to figure it out.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
